Android/android.py

```diff
@@ -175,7 +175,7 @@ def unpack_deps(host, prefix_dir):
     os.chdir(prefix_dir)
     deps_url = "https://github.com/beeware/cpython-android-source-deps/releases/download"
     for name_ver in ["bzip2-1.0.8-3", "libffi-3.4.4-3", "openssl-3.0.15-4",
-                     "sqlite-3.49.1-0", "xz-5.4.6-1"]:
+                     "sqlite-3.49.1-0", "xz-5.4.6-1", "zstd-1.5.7-0"]:
```

Comment:
Update from https://github.com/beeware/cpython-android-source-deps/pull/6:

```suggestion
                     "sqlite-3.49.1-0", "xz-5.4.6-1", "zstd-1.5.7-1"]:
```